src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -781,6 +781,18 @@ static int process_loader_partitions(DissectedPartition *esp, DissectedPartition
         assert(esp);
         assert(xbootldr);
 
+        /* If any paths in fstab look similar to our favorite paths for ESP or XBOOTLDR, we just exit
+         * early. We also don't bother with cases where one is configured explicitly and the other shall be
+         * mounted automatically. */
+
+        r = fstab_has_mount_point_prefix_strv(STRV_MAKE("/boot", "/efi"));
```

Comment:
Please just drop slash_boot_in_fstab() and friends then.

Reply:
yeah, good catch.